staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -183,6 +183,9 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.DurationVar(&s.StorageConfig.HealthcheckTimeout, "etcd-healthcheck-timeout", s.StorageConfig.HealthcheckTimeout,
 		"The timeout to use when checking etcd health.")
+
+	fs.Int64Var(&s.StorageConfig.LeaseReuseDurationSeconds, "etcd-lease-reuse-duration", s.StorageConfig.LeaseReuseDurationSeconds,
```

Comment:
I was wondering about switching to DurationVar (same as above).

That would be cleaner, but given that etcd only supports second-granularity of leases (and that is reflected in storage API, etc.), maybe that is actually fine.
It seems we recently used this pattern too in: https://github.com/kubernetes/kubernetes/pull/95533


So let's leave this Int64Var, but change the flag to:

"lease-reuse-duration-seconds"